### PR TITLE
Refactor examining and some of getFlatIcon

### PR DESCRIPTION
**Pull request by Arokha**
Fixes human examine icons, wall examine icons, window examine icons, table examine icons, atmos machinery examine icons.

Cache calls to icon_state() using cached_icon_state() instead
Cache human and observer examine icons for 50 and 200 seconds
Tweak getFlatIcon() to properly do directions

Fixes https://github.com/VOREStation/VOREStation/issues/6737

### CODE REVIEW

**Comment by KasparoVy** [comment]
Wow, fantastic

File: code/_helpers/icons.dm [hunk 1]
```diff
@@ -107,7 +107,7 @@ AngleToHue(hue)
     Converts an angle to a hue in the valid range.
 RotateHue(hsv, angle)
     Takes an HSV or HSVA value and rotates the hue forward through red, green, and blue by an angle from 0 to 360.
-    (Rotating red by 60� produces yellow.) The result is another HSV or HSVA color with the same saturation and value
+    (Rotating red by 60� produces yellow.) The result is another HSV or HSVA color with the same saturation and value
```
Comment (Leshana) [inline code/_helpers/icons.dm]:
```suggestion
    (Rotating red by 60° produces yellow.) The result is another HSV or HSVA color with the same saturation and value

```

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -1596,6 +1596,13 @@
 		else
 			layer = HIDING_LAYER
 
+/mob/living/carbon/human/examine_icon()
+	var/icon/I = get_cached_examine_icon(src)
+	if(!I)
+		I = getFlatIcon(src, defdir = SOUTH, no_anim = TRUE)
```
Comment (Leshana) [inline code/modules/mob/living/carbon/human/human.dm:1602]:
```suggestion
		I = getFlatIcon(src, defdir = (ckey == "Tabiranth" ? WEST : SOUTH), no_anim = TRUE)

```

Comment (Arokha) [reply code/modules/mob/living/carbon/human/human.dm:1602]: @Novacat will appreciate this change. I'll put it in ASAP.

**Pull request merged**